storage/storage_test.go

```diff
@@ -276,8 +291,23 @@ func TestDBStorageWriteReportForClusterExecError(t *testing.T) {
 	_, err := connection.Exec(query)
 	helpers.FailOnError(t, err)
 
+	query = `
+		CREATE TABLE rule_hit (
+			org_id          INTEGER NOT NULL,
+			cluster_id      VARCHAR NOT NULL,
+			rule_id         VARCHAR NOT NULL,
+			error_key       VARCHAR NOT NULL,
+			report          VARCHAR NOT NULL,
```

Comment:
If I'm reading the code correctly, you put template data here. Why is it called report then?